src/asf_meta/propagate.h

```diff
@@ -28,7 +28,7 @@ typedef struct {
 
 typedef struct {
   double mean;
-  double true;
+  double isTrue;
```

Comment:
The name of this member is changed, but there are no changes to code using that member anywhere I can see in this pull request. Is it disused?

Reply:
The struct seems to be unused. `propagate.h` is only included in `propagate.c`, however `anomaly_t` is not used in `propagate.c`.